pkg/deb/usr/lib/systemd/system/peerbanhelper.service

```diff
@@ -1,11 +1,14 @@
 [Unit]
 Description=PeerBanHelper Service
 After=network.target
+
 [Service]
 User=peerbanhelper
-WorkingDirectory=/etc/peerbanhelper
-ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
+WorkingDirectory=/usr/lib/peerbanhelper
```

Comment:
I think the wd should be the `$HOME` of user `peerbanhelper`, which means `/var/lib/peerbanhelper`. Anyway, it doesn't matter